packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/compose/models/HomeViewModel.kt

```diff
@@ -22,9 +26,10 @@ import kotlinx.coroutines.launch
 
 sealed interface HomeAction {
   class OpenApp(val url: String) : HomeAction
-  object RefetchRunningApps : HomeAction
-  object ResetRecentlyOpenedApps : HomeAction
+  data object RefetchRunningApps : HomeAction
+  data object ResetRecentlyOpenedApps : HomeAction
   class NavigateToCrashReport(val crashReport: DevLauncherErrorInstance) : HomeAction
+  data object ScanQRCode : HomeAction
```

Comment:
```suggestion
 object ScanQRCode : HomeAction
```

Reply:
Only added it because I hate ide warnings 😅